packages/mdc-select/mdc-select.scss

```diff
@@ -114,12 +114,8 @@
     background-position: left 10px center;
   }
 
-  .mdc-select__native-control,
-  .mdc-select__label {
-    @include mdc-typography(subtitle1);
-  }
-
   .mdc-select__native-control {
+    @include mdc-typography(subtitle1);
```

Comment:
I'm guessing this was removed from label to fix line-height, but do we not want other properties of this style?